test/specs/homework2.js

```diff
@@ -0,0 +1,113 @@
+// Due - Aug 19 (Friday eod)
+/**
+ * 
+ * Launch darksky.net
+ * Get all the timeline data in an array
+ * then print array in the console.
+ */
+ describe('Homework two', () => {
+    it('Print timeline data', async () => {
+
+        await browser.url('http://www.darksky.net')
+        await browser.pause(3000);
+
+        const hourClass = await $$("//div[@id='timeline']//div[@class='timeline']//div[@class='hours']//span[not(contains(@class,'hour'))]");
+        const array = [];
+
+        for (const hour of hourClass){
+            array.push(await hour.getText());
+        }
+
+        console.log(array);
+
+
+        
+    });
+/**
+ * Verify destination and check-in/ and check-out dates are as user selected
+ * 
+ * 1. Launch hotels.com
+ * 2. Type "man" in destination
+ * 3. Select "Manila" from auto-suggestion
+ * 4. Select Aug 20 to Sep-5 as check-in and check-out respectively
+ * 5. Click Search button
+ * 6. Verify destination has Manila
+ * 7. Verify check-in date in Aug-20
+ * 8. Verify check-out date in Sep-5
+ * 
+ * 
+ */
+
+    it.only('Verify destination and check-in/ and check-out dates are as user selected', async () => {
+        
+        // Launch hotels.com
+        await browser.url('https://www.hotels.com/');
+        await browser.pause(4000);
+        
+        // Type "man" in destination
+        await $("//button[@aria-label='Going to']").click();
+        await browser.pause(4000);
+        
+        await $('#destination_form_field').setValue('Man');
+        await browser.pause(4000);
+        
+        // Select "Manila"
+        const selectCity = await $$("//div[@class='truncate']//strong");
+        
+        for (const cityText of selectCity) {
+            const citySuggestion = await cityText.getText();
+            if (citySuggestion.toLowerCase().localeCompare('Manila'.toLowerCase()) === 0) {
+                await cityText.click();
+                break;
+            }
+        }
+        await browser.pause(4000);
+
+        await $('#date_form_field-btn').click();
+        await browser.pause(4000);
+
+        await $('//button[@data-stid="date-picker-paging"]').click();
+        await browser.pause(4000);
+        
+        // Select Aug 20 to Sep-5 as check-in and check-out respectively
+        const allAugDateElements = await $$('//h2[text()="August 2022"]/following-sibling::table//button[not(@disabled)]');
```

Comment:
Try to make more dynamic xpath by avoiding hard coding like `August 2022`. This xpath not gonna work next month.